cypress/e2e/settings/users/patron-block/patron-block-maximum-fee-fine-balance-removed-after-item-returned.cy.js

```diff
@@ -0,0 +1,295 @@
+import generateUniqueItemBarcodeWithShift from '../../../../support/utils/generateUniqueItemBarcodeWithShift';
+import InventoryInstances from '../../../../support/fragments/inventory/inventoryInstances';
+import PatronGroups from '../../../../support/fragments/settings/users/patronGroups';
+import Location from '../../../../support/fragments/settings/tenant/locations/newLocation';
+import Users from '../../../../support/fragments/users/users';
+import CirculationRules from '../../../../support/fragments/circulation/circulation-rules';
+import CheckInActions from '../../../../support/fragments/check-in-actions/checkInActions';
+import Checkout from '../../../../support/fragments/checkout/checkout';
+import ServicePoints from '../../../../support/fragments/settings/tenant/servicePoints/servicePoints';
+import InventoryInstance from '../../../../support/fragments/inventory/inventoryInstance';
+import getRandomPostfix from '../../../../support/utils/stringTools';
+import UsersOwners from '../../../../support/fragments/settings/users/usersOwners';
+import PaymentMethods from '../../../../support/fragments/settings/users/paymentMethods';
+import LoanPolicy from '../../../../support/fragments/circulation/loan-policy';
+import Conditions from '../../../../support/fragments/settings/users/conditions';
+import Limits from '../../../../support/fragments/settings/users/limits';
+import UsersSearchPane from '../../../../support/fragments/users/usersSearchPane';
+import UserLoans from '../../../../support/fragments/users/loans/userLoans';
+import LostItemFeePolicy from '../../../../support/fragments/circulation/lost-item-fee-policy';
+import UsersCard from '../../../../support/fragments/users/usersCard';
+
+describe('Patron Block: Maximum outstanding fee/fine balance', () => {
+  let originalCirculationRules;
+  const blockMessage = 'You have reached maximum outstanding fee/fine balance as set by patron group';
+  const patronGroup = {
+    name: 'groupToPatronBlock' + getRandomPostfix(),
+  };
+  const userData = {};
+  const itemsData = {
+    itemsWithSeparateInstance: [
+      { instanceTitle: `InstanceForDeclareLost ${getRandomPostfix()}` },
+      { instanceTitle: `InstanceForDeclareLost ${getRandomPostfix()}` },
+      { instanceTitle: `InstanceForDeclareLost ${getRandomPostfix()}` },
+      { instanceTitle: `InstanceForAgedToLost ${getRandomPostfix()}` },
+      { instanceTitle: `InstanceForAgedToLost ${getRandomPostfix()}` },
+    ],
+  };
+  const testData = {
+    userServicePoint: ServicePoints.getDefaultServicePointWithPickUpLocation('autotest fee/fine limit', uuid()),
+  };
+  const owner = {
+    body: {
+      owner: 'AutotestOwner' + getRandomPostfix(),
+      servicePointOwner: [
+        {
+          value: testData.userServicePoint.id,
+          label: testData.userServicePoint.name,
+        },
+      ],
+    },
+    data: {},
+  };
+  const lostItemFeePolicyBody = {
+    name: '1_lost' + getRandomPostfix(),
+    itemAgedLostOverdue: {
```

Comment:
this parameter "feesFinesShallRefunded":{"duration":6,"intervalId":"Months"} is missed